README.md

```diff
@@ -17,10 +17,12 @@ to operate with no central authority: managing transactions and issuing money
 are carried out collectively by the network. Dash Core is the name of the open
 source software which enables the use of this currency.
 
+Pre-Built Binary
+----------------
+
 For more information, as well as an immediately useable, binary version of
 the Dash Core software, see https://www.dash.org/get-dash/.
```

Comment:
Thanks for this PR (Am actually listening to your livestream in the background rn). However, we shouldn't be using this link anyway 🙈. I am happy to get better documentation when possible. 

```suggestion
the Dash Core software, see https://www.dash.org/downloads/.
```